### DISCUSSION TITLE
Expose `MakeIfThenElseCt` function in Python API of the Routing Solver

### DISCUSSION THREAD

**Discussion by arnabadas** [google/or-tools#3657] [Feature requests]
**What version of OR-Tools and what language are you using?**
Version: v9.5
Language: Python

**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**
Routing Solver

**What operating system (Linux, Windows, ...) and version?**
Windows 10

**What did you do?**
Use `MakeIfThenElseCt` function from `constraint_solver.h`

**What did you expect to see**
`MakeIfThenElseCt` should have been exposed in Python like C++ or Java API

**What did you see instead?**
`MakeIfThenElseCt` was not exposed in Python API

Make sure you include information that can help us debug (full error message, model Proto).

**Anything else we should know about your project / environment**
NA

**Comment by lperron** [comment]
You can use Element(condition, [if_false, if_true], target)
